test/tests/dict.py

```diff
@@ -194,3 +194,16 @@ def custom_keys():
 d.update({'c':3}, a=1, b=2)
 print sorted(d.items())
 
+# viewkeys / viewvalues / viewitems
+
+d = {}
+
+print list(d.viewkeys())
+print list(d.viewvalues())
+print list(d.viewitems())
+
+d['a'] = 1
+
+print list(d.viewkeys())
+print list(d.viewvalues())
+print list(d.viewitems())
```

Comment:
Can we add another check here that tests the difference between viewkeys and keys (and the others)?  ie mutate the dictionary and show that the views get updated but the non-views don't.